packages/plugins/plugin-table/src/model/update-listener.ts

```diff
@@ -31,6 +31,10 @@ export class CellUpdateListener {
     this.createInitialCellEffects();
   }
 
+  public setOnCellUpdate = (onCellUpdate: (cell: GridCell) => void): void => {
```

Comment:
can we refactor and simplify this, moving it into the model; only creating effects on a per object basis?